src/clientManager.js

```diff
@@ -18,7 +18,7 @@ ClientManager.prototype.init = function(identHandler, sockets) {
 	this.identHandler = identHandler;
 	this.webPush = new WebPush();
 
-	if (!Helper.config.public) {
+	if (!Helper.config.public && !Helper.config.ldap.enable) {
```

Comment:
Any reason to add this?

Reply:
This change broke loading existing ldap users on server restart.

It should disable fs.watch but still load existing json files from disk.